inspirehep/utils/cv_latex_html_text.py

```diff
@@ -203,19 +203,21 @@ def _get_publi_info(self):
                 out = ''
                 if 'journal_title' in field:
                     if isinstance(field['journal_title'], list):
-                        if not ('journal_volume' in field or
-                                'journal_issue' in field or
-                                'page_artid' in field or
-                                'doi' in self.record):
+                        if (('journal_volume' not in field or
+                                'journal_issue' not in field or
+                                ('page_start' not in field and
+                                 'artid' not in field)) and
+                                'doi' not in self.record):
```

Comment:
Definitely use a utility function + an explainer variable. I actually understood that for @ioannistsanaktsidis a paper is published if it has _any one_ of volume, issue, page_start/artid, which is what the tests are asserting.